src/tests/JIT/Regression/JitBlue/Runtime_39581/Runtime_39581.ilproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.IL">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+  <PropertyGroup>
+    <DebugType>None</DebugType>
+    <Optimize>True</Optimize>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <CLRTestBatchPreCommands Condition="'$(TargetOS)' == 'Windows_NT'"><![CDATA[
+$(CLRTestBatchPreCommands)
+set COMPlus_GCStress=C
+]]></CLRTestBatchPreCommands>
+    <BashCLRTestPreCommands Condition="'$(TargetOS)' == 'Linux'"><![CDATA[
+$(BashCLRTestPreCommands)
+export COMPlus_GCStress=C
```

Comment:
It's not safe in general to force GCStress because not all platforms support GCStress, or have the coredistools stress dependencies on the CI test machines. It's safe on Windows because, AFAICT, we always have coredistools on Windows (for x86/x64, where it is needed for GCStress), and stress is always supported.

Reply:
It's a temporary hack to make the test fail in CI without triggering a gcstress job.
I will undo this when I have the PR ready for review.